src/functions/get-packages.ts

```diff
@@ -30,17 +30,17 @@ const getPackageObject = (
 	const pkgRequiredFields: string[] = pkgTypeRequiredFieldMap[pkgType]
 
 	const propertiesFromPkgJson = pkgRequiredFields.reduce((fieldAcc, field) => {
-		const fieldValue = pkgObj[field]
+		const fieldValue = pkgObj.azureConfig?.[field]
```

Comment:
I would prefer the key called `azureDeployConfig` or `deployConfig`, since the settings are not just to do with azure but of the deployable package